core/src/edu/cornell/gdiac/temporary/GameplayController.java

```diff
@@ -31,10 +33,12 @@
 
 /**
  * Controller to handle gameplay interactions.
- *
+ * </summary>
+ * <remarks>
  * This controller also acts as the root class for all the models.
  */
 public class GameplayController {
+	int curframe;
```

Comment:
What happened with separating separate game updates (like the frames) from the GameplayController vs what makes the GameplayController? I think this was separated already.

This isn't the only case but if you don't see a need for this anymore then we can talk about this.

Reply:
I was going to change that after pulling from main again